policies/linux.consul.-.catalog.total_nodes.json

```diff
@@ -0,0 +1,27 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 2,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "metric": "consul.catalog.total_nodes",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a WARNING event when the number of total nodes drops.",
```

Comment:
Could you provide a more user friendly description of what this policy is indicating to a user receiving the alert? Any action which should be taken?

Reply:
@Grumblesmith I've commited the requested changes to the branch